Gemfile

```diff
@@ -46,8 +46,8 @@ gem 'dotenv-rails', '~> 2.1'
 gem 'rails_12factor', group: :production
 gem 'ood_cluster', '~> 0.0.1'
 gem 'ood_appkit', '~> 1.0'
-gem 'pbs', '~> 2.2.1'
 gem 'jquery-datatables-rails', '~> 3.3'
+gem 'ood_core', github: 'OSC/ood_core', branch: 'master'
```

Comment:
Note that we use Bundler version 1.x and when I install `ood_core` using the Github keyword instead of an HTTPS URL I get the following warning:
```
The git source `git://github.com/OSC/ood_core.git` uses the `git` protocol, which transmits data without encryption. Disable this warning with `bundle config git.allow_insecure true`, or switch to the `https` protocol to keep your data secure.
```
I imagine that this is not the final state of the Gemfile for this release, but in general we should prefer using HTTPS URLs even if they are longer.

Reply:
A bit of discussion on the topic from the Bundler people: https://bundler.io/v1.2/git.html#security

Reply:
>Note that the :github shortcut translates to a git:// URL in pre-2.0 versions. 

Too bad we can't upgrade Bundler to 2.0 as it requires ruby gems >= 3.0.0

```
efranz@webtest04:~/awesim/dev/activejobs (perf-improv)$ gem --version
2.6.14
```